ACT-reqs.md

```diff
@@ -0,0 +1,84 @@
+
+The ACT Framework will standardize those aspects all rules have in common that ensure them to be reliable, implementable by different ATTs and transparent for the users of this tool. By standardizing this, the ACT Taskforce aims to promote a common understanding of how accessibility can be tested for certain technologies.
+
+Simply put, the ACT Framework is a standard on how to write test rules. Rules written in conformance to the ACT Framework specification are referred to as ACT Rules. These in turn are test procedures for accessibility, that can be implemented by ATTs or used for testing of accessibility.
+
+2. Requirements
+---------------
+
+### 2.1 Ensure Readability
+
+The ACT Framework should ensure that the rules are readable by both tool developers and less-technical users. For instance, the ACT Framework may discourage the use of pseudo code in favor of plain English.
+
+### 2.2 Ensure Justification
+
+The ACT Framework should ensure that ACT Rules are well grounded in the requirements they test. It is insufficient for an ACT Rule to merely claim something violates a requirement. The ACT Framework has to ensure that ACT Rules document why its claims are a valid interpretation of the requirement.
+
+### 2.3 Ensure Consistency
+
+The ACT Framework has to ensure that ACT Rules can be implemented in a way that provides consistent results. An ACT Rule must include a way to test the implementation of that rule.
+
+### 2.4 Ensure Measurable Accuracy
+
+The ACT Framework ensures that ACT Rules can be tested for real world accuracy. False positives happen in every rule, as there are endless different ways to use web technologies. The ACT Framework ensures that common false positives can be identified and resolved. This also gives insight into how often a rule may give incorrect results.
+
+### 2.5 Ensure Evolution
+
+As web technologies change, the rules for testing them should change as well. Because of this it is important that ACT Framework outlines a way to deal with that. Rules should be dated and versioned, so that users of rules can understand that a rule needs to be updated. A possible approach to this could be semantic versioning, although there are many other possibilities.
+
+### 2.6 Existing Rulesets can be Transformed to ACT Rules
+
+The ACT Framework must ensure that existing rulesets can be updated to meet the ACT Framework. Existing work in this field should be the bases for the ACT Framework to ease the adoption. At least one ruleset must be adopted to the ACT Framework during the development process.
```

Comment:
'bases' should be 'basis'
I'm confused by the last statement, "At least one ruleset must be adopted to the ACT Framework during the development process."  If the ACT framework is to be a Recommendation, doesn't it require two implementations?